integration_tests/configs/default.jsonnet

```diff
@@ -0,0 +1,86 @@
+{
+  dotenv: '../../scripts/.env',
```

Comment:
Should we replace `dotenv` trick with jsonnet expressions as well?

Reply:
not sure if you bother env parser itself? since read env logic is [fixed](https://github.com/crypto-com/pystarport/pull/52/commits/e1f67e2424415c08f637a20203b40dacbebaf8c5)

Reply:
I just think it's simple enough with jsonnet, and potentially deprecate dotenv feature in the future, so keep only one way to do the same thing.

Reply:
> not sure if you bother env parser itself? since read env logic is [fixed](https://github.com/crypto-com/pystarport/pull/52/commits/e1f67e2424415c08f637a20203b40dacbebaf8c5)

I see, do you mean dotenv provides env reading ability, which doesn't exist in jsonnet approach?

Reply:
yes